### DISCUSSION TITLE
how to open multiple Ports on spostal mail server

### DISCUSSION THREAD

**Discussion by KyliWalker** [postalserver/postal#2924] [Help with using Postal]
Dear Team how open multiple Port on postal mail server my confuartion is 

smtp_server:
  host: 5.196.8.191
  port: 587
  bind_address: '::'
  tls_enabled: false
  tls_certificate_path: # Defaults to config/smtp.cert
  tls_private_key_path: # Defaults to config/smtp.key
  tls_ciphers:
  ssl_version: SSLv23
  proxy_protocol: false
  log_connect: true
  strip_received_headers: false

**Comment by willpower232** [comment]
You can accomplish that with some iptables rules or a customer docker compose file however it is likely that the ports you want to use are expecting TLS by default which is not supported by Postal. All connections must start plain and upgrade to TLS with STARTTLS.

You are almost always better off by running Postal on port 25 only.

**Comment by KyliWalker** [comment]
Hey sir 

In vim /opt/postal/config/postal.yml

I am using below confguartion But i am not able to send more then 5000 email per day 

can suggest me confguartion so i can fix it 

version: 2

postal:
  web_hostname: dg.xxx,com
  web_protocol: https
  smtp_hostname: dg.xxx,com

main_db:
  host: 127.0.0.1
  username: root
  password: postal
  database: postal

message_db:
  host: 127.0.0.1
  username: root
  password: postal
  prefix: postal

smtp_server:
  default_bind_address: "::"

dns:
  # Specify the DNS records that you have configured. Refer to the documentation at
  # https://github.com/atech/postal/wiki/Domains-&-DNS-Configuration for further
  # information about these.
  mx_records:
    - mx.dg.xxx,com
  spf_include: spf.xxx,com
  return_path_domain: rp.dg.xxx,com
  route_domain: routes.xxx,com
  track_domain: track.xxx,com

  
smtp:
  # Specify an SMTP server that can be used to send messages from the Postal management
  # system to users. You can configure this to use a Postal mail server once the
  # your installation has been set up.
  host: 127.0.0.1
  port: 2525
  username: # Complete when Postal is running and you can
  password: # generate the credentials within the interface.
  from_name: Postal
  from_address: dg.xxx,com

Mysql:
  innodb_buffer_pool_size = 2048M
  innodb_flush_log_at_trx_commit = 2
  max_connections   = 8000
  wait_timeout    = 31536000
  max_allowed_packet  = 256M

workers:
  quantity: 50
  threads: 80

main_db:
  pool_size: 150

message_db:
  pool_size: 50

**Reply by willpower232** [reply]
it sounds like you need to scale some more workers in so you have more processes working on the queue.

**Comment by KyliWalker** [comment]
can you share how many i need to put here 
workers:
quantity: 50
threads: 80

**Reply by willpower232** [reply]
I don't think these numbers have any effect, you would need to add more workers with either the `--scale` argument or a docker-compose.override.yml

**Reply by KyliWalker** [reply]
How can i increase sending speed with current version of server [Postal](https://postalserver.io/) 3.3.4
